validation/github/standardized-repository-settings/main.tf

```diff
@@ -0,0 +1,15 @@
+terraform {}
+
+provider "github" {}
+
+module "standardized_repository_settings" {
+  source = "../../../modules/github/standardized-repository-settings"
+
+  repository_name = "test"
+
+  repository_description = "a description of a repo"
+
+  repository_visibility = "private"
+
+  required_status_checks = ["test", "ci"]
```

Comment:
What happens if this list is empty? All our repos don't have ci pipelines, especially not the legacy private ones that just contains scripts.

Reply:
If this list is set to empty, then no status checks are required (verified in documentation).